sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/StandardMetricConstants.cs

```diff
@@ -27,5 +29,7 @@ internal static class StandardMetricConstants
         internal const string CloudRoleNameKey = "cloud/roleName";
         internal const string CloudRoleInstanceKey = "cloud/roleInstance";
         internal const string MetricIdKey = "_MS.MetricId";
+
+        internal static  ReadOnlySpan<string> SyntheticUserAgentValues => new string[] { "AlwaysOn", "AppInsights", "search", "spider", "crawl", "Bot", "Monitor", "Gomez", "Pingdom" };
```

Comment:
it'd be nice how this list was come up with. Particularly "AppInsights".

Reply:
hmmm. Let me follow up with Jackson on this. 
This is the list that was in our spec, but it differs from our legacy sdk.
https://github.com/microsoft/ApplicationInsights-dotnet/blob/99118c924d3880f3853c0b1c9ebe67289073dbfa/examples/.NET%20Config%20File/2.10.0_ApplicationInsights.config#L12

Reply:
I don't think this should block the PR. Any changes should come through the spec and we can update this list as needed

Reply:
That is just example of the extended list, not the default followed in ApplicationInsights. The actual list is "search|spider|crawl|Bot|Monitor|AlwaysOn". 

Also, the above is not covering ApplicationInsights WebTests.
They are covered by separate headers (different from user-agent) https://github.com/microsoft/ApplicationInsights-dotnet/blob/main/NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/SyntheticTelemetryInitializer.cs#L42-L43
Do we know if ApplicationInsights WebTests send a useragent from the above list? if not, please make sure the spec is fixed.

Reply:
Ohhhhh, we have different Synthetic TelemetryInitializers in the legacy sdk. I missed this.

- https://github.com/microsoft/ApplicationInsights-dotnet/blob/main/WEB/Src/Web/Web/SyntheticUserAgentTelemetryInitializer.cs
- https://github.com/microsoft/ApplicationInsights-dotnet/blob/main/NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/SyntheticTelemetryInitializer.cs